pkg/health/server/prober.go

```diff
@@ -85,6 +85,16 @@ func getPrimaryIP(node *ciliumModels.NodeElement) string {
 	return node.PrimaryAddress.IPV6.IP
 }
 
+func getHealthIP(node *ciliumModels.NodeElement) string {
+	if node.HealthEndpointAddress == nil {
```

Comment:
`if node == nil || ...`

Reply:
This is only used while iterating through a map of nodes which we populate with non-nil objects.. is there a path in mind that you're worried about here?

Reply:
Probably not, just worried about if something sets the map with `nil` and we only find out through a panic